app/commands/github/task/sync_tasks.rb

```diff
@@ -0,0 +1,22 @@
+module Github
+  class Task
+    class SyncTasks
+      include Mandate
+
+      def call
+        issues.each do |issue|
+          SyncTask.(issue)
+        rescue StandardError => e
+          Rails.logger.error "Error syncing task for issue #{issue.repo}/#{issue.number}: #{e}"
+        end
+
+        nil
+      end
+
+      private
+      def issues
+        Github::Issue.left_joins(:labels)
```

Comment:
```suggestion
        Github::Issue.includes(:labels)
```

If you're just looking to avoid `n+1s` as opposed to using the join in SQL, you're better using `includes` than `joins`, as it simplifies the SQL but has the same n+1 guarding. (It might be `include` - I can never remember)

Reply:
Yep, works well